proposals/20210524-versioning-and-release-of-the-libs-artifacts.md

```diff
@@ -0,0 +1,97 @@
+# Versioning and release process of the libs artifacts
+
+## Summary
+
+This proposal is meant as a follow-up to the [OSS Libraries Contribution](https://github.com/falcosecurity/falco/blob/master/proposals/20210119-libraries-contribution.md) one.
+
+In particular, it outlines the versioning and the release process of the `falcosecurity/libs` repository (containing libscap, libsinsp, and the two Falco drivers).
+
+## Motivation
+
+According to the [OSS Libraries Contribution Plan](https://github.com/falcosecurity/falco/blob/master/proposals/20210119-libraries-contribution.md), in
+January/February 2021 the libscap, libsinsp, and the two Falco drivers have been contributed to the CNCF ([blog post](https://falco.org/blog/contribution-drivers-kmod-ebpf-libraries), [announcement](https://www.cncf.io/blog/2020/01/08/toc-votes-to-move-falco-into-cncf-incubator)).
+
+Falco heavily relies on those libraries and on the two drivers to work.
+Currently, Falco fetches them via [git commit](https://github.com/falcosecurity/falco/blob/da7279da1dc240e8f242fa33b2a73eff178a5c87/cmake/modules/falcosecurity-libs.cmake#L23) at build time.
+
+This git reference is - at the moment - the only notion of "version" existing for the libraries.
+
+One of the reasons for the contribution of the libraries was to enable people to benefit from those libraries by using them directly in their OSS projects.
+
+For this reason, the need of giving to the libraries a clear versioning scheme arises.
+
+The [OSS Libraries Contribution Plan](https://github.com/falcosecurity/falco/blob/master/proposals/20210119-libraries-contribution.md) had already identified at the time the artifacts regarding `falcosecurity/libs` (points 8-9 for [libsinsp](https://github.com/falcosecurity/falco/blob/master/proposals/20210119-libraries-contribution.md#libsinsp), points 8-9 for [libscap](https://github.com/falcosecurity/falco/blob/master/proposals/20210119-libraries-contribution.md#libscap)).
+
+This proposal intends to extend on points 10-12 for the libsinsp and libscap plans to graduate them to "Offical Support" (see the [evolution process](https://github.com/falcosecurity/evolution#official-support)).
+
+## Goals
+
+- Define a versioning mechanism for the artifacts
+- Outline an automated release process of the artifacts
+
+## Non-Goals
+
+- Define the artifacts
```

Comment:
I don't understand how you can have two goals that say

```
Define a versioning mechanism for the artifacts
```

and

```
Outline an automated release process of the artifacts
```

Without doing the `Define the artifacts` part of it.

Reply:
I'll talk more about this proposal during the community call today anyway :)